src/Workspaces/Core/Portable/CodeStyle/CodeStyleOption.cs

```diff
@@ -7,7 +7,11 @@
 using System.Xml.Linq;
 using Microsoft.CodeAnalysis.Diagnostics;
 
+#if CODE_STYLE
```

Comment:
These are the core namespace changes - all of these are temporary and can be reverted once we move to a common AnalyzerConfigOptions based infrastructure for analyzer execution in Workspaces.